Specs/addDefaultMatchers.js

```diff
@@ -416,6 +432,17 @@ define([
         };
     }
 
+    // converts x and y coordinates such that after updating for drawing buffer position, the correct coordinates will be used
+    function pickPrimitiveEqualsWrapper(actual, expected, x, y, width, height) {
+        width = width || 3;
+        height = height || width || 3;
+        x = x || 0;
+        y = y || 0;
+        var adjustedX = x + ((width - 1) * 0.5);
+        var adjustedY = -1 * (y + ((height - 1) * 0.5) - actual._context.drawingBufferHeight);
+        return pickPrimitiveEquals(actual, expected, adjustedX, adjustedY, width, height);
```

Comment:
I'm not completely following the conversion process here. Can the test be modified instead?